starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,64 @@
 // Constructor
-// function Chronometer() {
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId;
+  this.minutes;
+  this.seconds;
+  this.milliSeconds;
+}
 
-// }
+Chronometer.prototype.startClick = function () {    
+  var that = this;
 
-// Chronometer.prototype.startClick = function () {
+  this.intervalId = setInterval(function () {   // Every 10 milisec counter increment by 1
+    that.currentTime += 1;
+    console.log(that.currentTime)
+  }, 10);
+};
 
-// };
+Chronometer.prototype.setMinutes = function () {
+  return this.minutes = Math.floor(Math.floor(this.currentTime / 100) / 60);  //calculated minutes from milisec
+};
```

Comment:
Don't think we need to use `Math.floor` multiple times here.  Just the outer ones should be enough.  Also, have a look into the `%` operator.  This operator return the remainder after dividing by a given number.  So to calculate the times, I think this is more efficient:
```
this.seconds = (this.currentTime % this.minutes) / 100 
// Get remainder of milliseconds after minutes are calculated;
// Divide by 100 to calculate seconds
```